app/controllers/ops_controller/settings/common.rb

```diff
@@ -355,6 +355,14 @@ def settings_update_email_verify
     javascript_flash
   end
 
+  def update_server_zone(server)
+    zone = Zone.find_by(:name => @edit[:new][:server][:zone])
+    return true if zone.nil? || server.zone == zone
```

Comment:
`return true if zone.nil?`

should we silently ignore attempts to set a zone that doesn't exist?
(OTOH it does match the current behaviour)

Reply:
Yeah, I tried to not change this logic as much as possible, just wanted to break it out so it was more easily testable.